### ISSUE TITLE
Support for installation via Gemfile.

### ISSUE THREAD

**Issue by lolgear** [johnotander/pixyll#354]
So, it would be nice if theme could be installed by Gemfile

```
gem 'jekyll-theme-pixyll' 
```

**Comment by ashawley** [comment]
I worked on this in #334.  Please add your 👍  to #333.

**Comment by lolgear** [comment]
@ashawley done! Do you need any help to support gems?

**Comment by ashawley** [comment]
Yeah, my gem patch needs people to review it.  If you are familiar with gems and would want to give it a spin, that would be helpful.  http://guides.rubygems.org/make-your-own-gem/

**Comment by johnotander** [comment]
Thanks for the hard work on this @ashawley 💟 ! Hoping to have time in the next week or so to review and publish to rubygems.

**Comment by ashawley** [comment]
I am going to go through the jekyll Quick start and enable pixyll as the theme and see if there are issues.  

Building a gem means the theme needs to run on a variety of versions of ruby, jekyll, and so on.  This is a lot different than using the theme as a repo with pinned dependencies.  Might an automated test suite that runs in a variety environments be useful to keep track of whether the gem is working or not?  I'm going to propose a Travis config be added in a sepearate PR.  Might this conflict with downstream git fork users?

**Comment by ashawley** [comment]
I found other issues, but @johnotander quickly accepted my PR containing an initial Travis build. 
 I'll be able to grow the Travis build for the gem branch.

**Comment by claudiuconstantin** [comment]
Any news here?

**Comment by ashawley** [comment]
I need to further study a few more issues.  See my last comment in #334.

**Comment by ashawley** [comment]
I'm picking this up again.  I was about to close this issue, but wanted to discuss one aspect.  Should we call the gem/theme `pixyll` or is there a convention that it should be `jekyll-theme-pixyll`:

```
gem 'jekyll-theme-pixyll' 
```

I had planning to name it "pixyll", but am happy to open the bikeshedding to everyone.  Please comment, but also vote:

**Comment by ashawley** [comment]
# Motion to name the theme: `jekyll-theme-pixyll`

**Comment by ashawley** [comment]
# Motion to name the theme: `pixyll`

**Comment by plttn** [comment]
I'd lean towards `jekyll-theme-pixyll` simply because it seems to be convention on rubygems.

**Comment by claudiuconstantin** [comment]
What about `pixyll-Jekyll`?

**Comment by ashawley** [comment]
I agree.  The `jekyll-theme-*` prefix does seem to be a convention but there are exceptions, including [minima](https://rubygems.org/gems/minima).

**Comment by johno** [comment]
My 2c is that if `pixyll` is available we should use it! But I'll defer to @ashawley and @plttn.
